airflow/dags/ingestion/ask-astro-load.py

```diff
@@ -328,7 +330,21 @@ def extract_astro_blogs():
 
         return [df]
 
-    @task(trigger_rule="none_failed")
+    @task(trigger_rule=TriggerRule.NONE_FAILED)
+    def get_cached_or_extract_cosmos_docs():
+        from include.tasks.extract import cosmos_docs
+
+        parquet_file_path = "include/data/astronomer/cosmos/cosmos_docs.parquet"
+
+        try:
+            df = pd.read_parquet(parquet_file_path)
+        except Exception:
```

Comment:
Ideally it would be nice to use a more specific exception here like `FileNotFoundError` or an equivalent. Other exceptions should fail outside of attempting to read the file.